web/apps/apps.go

```diff
@@ -294,15 +289,15 @@ func findAccountsToDelete(instance *instance.Instance, slug string) ([]deletionE
 		if err == nil && msg.Slug == slug && msg.Account != "" {
 			acc := &account.Account{}
 			if err := couchdb.GetDoc(instance, consts.Accounts, msg.Account, acc); err == nil {
-				entry := deletionEntry{acc, t}
+				entry := account.CleanEntry{Account: acc, Trigger: t}
```

Comment:
I have not understood why here we pass the trigger whereas in the other method, we do not pass the trigger.

Reply:
When we uninstall a konnector, we remove the accounts, and we also delete the associated triggers. When the stack is destroyed, we don't try to find which trigger is related to which account, we just clean the accounts, and all the triggers are deleted in bulk.